Tests/scripts/secrets.py

```diff
@@ -0,0 +1,287 @@
+import io
+import os
+import re
+import math
+import json
+import string
+import PyPDF2
+import validate_files_structure
```

Comment:
from validate_files_structure import run_git_command
it's better practice to import just the things you need 
and separate that from the other imports

relevant for lines 69,73